contracts/src/OPSuccinctDisputeGame.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {OPSuccinctL2OutputOracle} from "./OPSuccinctL2OutputOracle.sol";
+import {CWIA} from "@solady/utils/legacy/CWIA.sol";
+import {LibBytes} from "@solady/utils/LibBytes.sol";
+import {ISemver} from "@optimism/src/universal/ISemver.sol";
+import {IDisputeGame} from "@optimism/src/dispute/interfaces/IDisputeGame.sol";
+import {Claim, GameStatus, GameType, GameTypes, Hash, Timestamp} from "@optimism/src/dispute/lib/Types.sol";
+import {GameNotInProgress, OutOfOrderResolution} from "@optimism/src/dispute/lib/Errors.sol";
+
+contract OPSuccinctDisputeGame is ISemver, CWIA, IDisputeGame {
+    using LibBytes for bytes;
+
+    /// @notice The address of the L2 output oracle contract.
+    address internal immutable l2OutpoutOracle;
+
+    /// @notice The timestamp of the game's global creation.
+    Timestamp public createdAt;
+
+    /// @notice The timestamp of the game's global resolution.
+    Timestamp public resolvedAt;
+
+    /// @notice Returns the current status of the game.
+    GameStatus public status;
+
+    /// @notice Semantic version.
+    /// @custom:semver v1.0.0-rc2
+    string public constant version = "v1.0.0-rc2";
+
+    constructor(address _l2OutpoutOracle) {
+        l2OutpoutOracle = _l2OutpoutOracle;
+    }
+
+    ////////////////////////////////////////////////////////////
+    //                    IDisputeGame impl                   //
+    ////////////////////////////////////////////////////////////
+
+    function initialize() external payable {
+        createdAt = Timestamp.wrap(uint64(block.timestamp));
+        status = GameStatus.IN_PROGRESS;
+
+        (uint256 l2BlockNumber, uint256 l1BlockNumber, bytes memory proof) =
+            abi.decode(extraData(), (uint256, uint256, bytes));
+
+        OPSuccinctL2OutputOracle(l2OutpoutOracle).proposeL2Output(
+            rootClaim().raw(), l2BlockNumber, l1BlockNumber, proof
+        );
+
+        this.resolve();
+    }
+
+    /// @notice Getter for the game type.
+    /// @dev The reference impl should be entirely different depending on the type (fault, validity)
+    ///      i.e. The game type should indicate the security model.
+    /// @return gameType_ The type of proof system being used.
+    function gameType() public pure returns (GameType) {
+        return GameTypes.CANNON;
```

Comment:
Is this correct? We aren't using CANNON here right?

Reply:
No but there is nothing matching currently. But we can add `OP_SUCCINCT`, see this: https://discord.com/channels/1244729134312198194/1260611879391461541/1328399228267008042

Reply:
Link to the PR that is adding OP_SUCCINCT GameType in this PR. Can you also make an issue on the repository for it?